QGL/PulsePrimitives.py

```diff
@@ -672,16 +673,17 @@ def CNOT_CR(controlQ, targetQ, **kwargs):
 
     if edge.isforward(controlQ, targetQ):
         # control and target for CNOT and CR match
-        return ZX90_CR(controlQ, targetQ, **kwargs) + \
+        seq = ZX90_CR(controlQ, targetQ, **kwargs).seq + \
```

Comment:
The trailing `.seq` is one bit of ugliness that I'd like to get rid of. Anyone have a suggestion?